cmd/kube-proxy/app/server_windows.go

```diff
@@ -85,6 +84,10 @@ func newProxyServer(config *proxyconfigapi.KubeProxyConfiguration, cleanupAndExi
 	if err != nil {
 		return nil, err
 	}
+
+	nodeIP := detectNodeIP(client, hostname, config.BindAddress)
+	klog.Infof("Detected node IP %s", nodeIP.String())
```

Comment:
This package seems to be using structured logging. Could we make the new log statements structured?

Reply:
good catch!  this landed after i initially created this pr